botocore/__init__.py

```diff
@@ -60,8 +60,17 @@ def emit(self, record):
 
 BOTOCORE_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+
 # Used to specify anonymous (unsigned) request signature
-UNSIGNED = object()
+class UNSIGNED(object):
+    def __copy__(self):
+        return self
+
+    def __deepcopy__(self, memodict={}):
```

Comment:
Not sure if we should be instantiating a dictionary in the method signature. Is this a special case for `__deepcopy__()`?